src/DocumentDB.ChangeFeedProcessor/DocumentDB.ChangeFeedProcessor.csproj

```diff
@@ -62,11 +62,11 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="1.9.1" />
+    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.0.0-preview" />
```

Comment:
should probably be preview2?

Reply:
Should almost definitely be preview2, it's where the signature changes occurred.